docs/source/cuvs_bench/index.rst

```diff
@@ -0,0 +1,702 @@
+~~~~~~~~~~
+cuVS Bench
+~~~~~~~~~~
+
+cuVS bench provides a benchmarking tool for various ANN search implementations. It's especially suitable for comparing GPU implementations as well as comparing GPU against CPU.
+
+- `Installing the benchmarks`_
+
+  * `Conda`_
+
+  * `Docker`_
+
+
+Installing the benchmarks
+=========================
+
+There are two main ways pre-compiled benchmarks are distributed:
+
+- `Conda`_ For users not using containers but want an easy to install and use Python package. Pip wheels are planned to be added as an alternative for users that cannot use conda and prefer to not use containers.
+- `Docker`_ Only needs docker and [NVIDIA docker](https://github.com/NVIDIA/nvidia-docker) to use. Provides a single docker run command for basic dataset benchmarking, as well as all the functionality of the conda solution inside the containers.
+
+Conda
+-----
+
+.. code-block:: bash
+
+   mamba create --name raft_ann_benchmarks
+   conda activate raft_ann_benchmarks
+
+   # to install GPU package:
+   mamba install -c rapidsai -c conda-forge -c nvidia raft-ann-bench=<rapids_version> cuda-version=11.8*
+
+   # to install CPU package for usage in CPU-only systems:
+   mamba install -c rapidsai -c conda-forge  raft-ann-bench-cpu
```

Comment:
btw - feel like we should be consistent about either using conda or mamba (so if we don't change the `conda install` instructions, we should change this)

Reply:
I agree, that was an accident- I missed that one.